components/cam/cime_config/config_compsets.xml

```diff
@@ -353,6 +353,52 @@
     <lname>2000_CAM5%AQUA_SLND_SICE_DOCN%AQPFILE_SROF_SGLC_SWAV</lname>
   </compset>
 
+  <!-- MMF aquaplanet compsets -->
+  <compset>
+    <alias>F-MMF1-AQP1</alias>
+    <lname>2000_CAM5%MMF1-AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV</lname>
```

Comment:
Did you set the values related with AQP[1-10] in config_components.f90?

Reply:
yes, the "DOCN" part is what sets the SST pattern associated with the number.